src/Schema/Tables/Secrets.sql

```diff
@@ -0,0 +1,7 @@
+﻿CREATE TABLE [dbo].[Secrets]
+(
+	[UserId] INT NOT NULL, 
+    [Password] NVARCHAR(50) NOT NULL
```

Comment:
@Anastasiia37 @Kropisha @smoukiDev @FSuhov @ArteIIo @DenyaAbramenkov @MaksimTsedov
please double check your formatting with any of such sites.  google - 'sql formatter'
http://www.dpriver.com/pp/sqlformat.htm

But make sure that syntax and code convention is same as was initial